.grype.yaml

```
@@ -10,3 +10,7 @@ ignore:
   - vulnerability: CVE-2021-22570
     package:
       name: google.golang.org/protobuf
+  # We can ignore this CVE since it's related to QUIC which we don't use.
+  - vulnerability: GHSA-cvx7-x8pj-x2gw
```

```suggestion
  - vulnerability: CVE-2025-47950
```

(it’s strange to have a comment mentioning “CVE” and then a GHSA)